### DISCUSSION TITLE
Xbox wireless adapter compatibility

### DISCUSSION THREAD

**Discussion by Sumpftier** [ndeadly/MissionControl#344] [Ideas]
I would love to use my xbox wireless adapter with my gen1 elite controller on switch. I remember that cathery had problems with that and could never implement it into syscon. How big are the chances for future compatibility?

**Comment by parkerlreed** [comment]
syscon would be where this is implemented. MissionControl doesn't handle wired accessories. 

Either way the wireless dongle is its own damn WiFi adapter. Implementing it anywhere would be a huge pain in the ass and not worth the trouble. Most people will just have the newer controller which supports Bluetooth.

**Comment by ndeadly** [comment]
Mission control will probably support USB at some point in the future. At this point it might be possible to add compatibility.

Doesn't sys-con already support Xbox 360 controllers via the wireless adapter? Do you know whether cathery's issue was with the adapter or the elite controller itself? If you can give me any idea of what the actual issue is I maybe be able to solve it in sys-con for you. I recently made some modifications to sys-con to improve controller support for another user as a stopgap until I get around to adding USB myself.

**Reply by shawn8888** [reply]
wireless adapter with xbox 360 works with sys-con for me!

**Reply by Sumpftier** [reply]
> Mission control will probably support USB at some point in the future. At this point it might be possible to add compatibility.
> 
> Doesn't sys-con already support Xbox 360 controllers via the wireless adapter? Do you know whether cathery's issue was with the adapter or the elite controller itself? If you can give me any idea of what the actual issue is I maybe be able to solve it in sys-con for you. I recently made some modifications to sys-con to improve controller support for another user as a stopgap until I get around to adding USB myself.

Man sry I didnt see your response until now... Cathery had many problems with both the new and the old xbox one wireless adapters. They said that its possible to get them working but requires too much work they werent wiling to put into.

**Comment by shawn8888** [comment]
The latest sys-con 0.6.4 doesn't support 13.1 yet, does it? 
Otherwise I would love to give it a try with my wired and wireless xbox 360 controllers.

**Reply by ndeadly** [reply]
Not sure whether it requires an update or not. Try it and see? I might be able to update/recompile it to work if the dev hasn't done it yet.

**Reply by shawn8888** [reply]
I just test it and it works!
Firmware 13.1
Atmosphere: 1.2.1
Sys-con: 0.6.4
Tested XBOX 360 controllers, both wired and wireless (with a wireless adapter). Both work great!
